### DISCUSSION TITLE
Build error when building with cargo cross and x86_64-pc-windows-gnu target

### DISCUSSION THREAD

**Discussion by inosms** [Thinkofname/steamworks-rs#29] [General]
When building a project depending on this library using cargo cross for the x86_64-pc-windows-gnu target the build fails and a very long error is printed. 

The solution for this is to remove the type definitions under after `#if defined( _WIN32 )` in `steamtypes.h`. As I don't have any Windows PC I do not know whether the resulting binary works yet, but I wanted to open this issue in case anybody else has the same problem. 

Edit: the binary works

**Comment by Thinkofname** [comment]
`steamtypes.h` is from the steamworks sdk so it isn't in my control to fix it. I've avoided the windows-gnu backend after encountering some hard to debug issues when combined with steamworks awhile back.

**Comment by inosms** [comment]
Ah, yes of course!
I simply wanted to note that cross-compiling for windows is in fact possible when editing `steamtypes.h` just in case anyone else experiences the same problem - I guess this issue can then be closed?
